pom.xml

```diff
@@ -185,7 +185,8 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <debug>false</debug>
+                            <!-- TODO(ja-lib) can this be causing the warning? -->
+<!--                            <debug>false</debug>-->
```

Comment:
It's my understanding that the warning is coming from a dependency class that isn't compiled with `parameters` so it's falling back to debug. See https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-6.x#parameter-name-retention
 
When I compile our code with `<parameters>true</parameters>` the classes that that log gets printed for are all JPARepository classes:

com.box.l10n.mojito.service.commit.CommitToPullRunRepository
com.box.l10n.mojito.service.pullrun.PullRunAssetRepository
com.box.l10n.mojito.service.pullrun.PullRunTextUnitVariantRepository
com.box.l10n.mojito.service.assetTextUnit.AssetTextUnitRepository
com.box.l10n.mojito.service.tm.TMTextUnitVariantCommentRepository
com.box.l10n.mojito.service.assetExtraction.AssetTextUnitToTMTextUnitRepository
com.box.l10n.mojito.service.branch.BranchRepository
com.box.l10n.mojito.service.assetcontent.AssetContentRepository
com.box.l10n.mojito.service.commit.CommitToPushRunRepository
com.box.l10n.mojito.service.pushrun.PushRunRepository
com.box.l10n.mojito.service.pushrun.PushRunAssetRepository
com.box.l10n.mojito.service.pushrun.PushRunAssetTmTextUnitRepository
com.box.l10n.mojito.service.security.user.UserRepository
com.box.l10n.mojito.service.security.user.AuthorityRepository
com.box.l10n.mojito.service.commit.CommitRepository
com.box.l10n.mojito.service.translationkit.TranslationKitRepository
com.box.l10n.mojito.service.translationkit.TranslationKitTextUnitRepository
com.box.l10n.mojito.service.branch.BranchStatisticRepository
com.box.l10n.mojito.service.screenshot.ScreenshotRepository
com.box.l10n.mojito.service.screenshot.ScreenshotTextUnitRepository
com.box.l10n.mojito.service.thirdparty.ThirdPartyScreenshotRepository
com.box.l10n.mojito.service.thirdparty.ThirdPartyTextUnitRepository
com.box.l10n.mojito.service.gitblame.GitBlameRepository
com.box.l10n.mojito.service.branch.BranchTextUnitStatisticRepository
com.box.l10n.mojito.service.branch.notification.BranchNotificationRepository
com.box.l10n.mojito.service.pluralform.PluralFormForLocaleRepository
com.box.l10n.mojito.service.repository.statistics.StatisticsScheduleRepository
com.box.l10n.mojito.service.thirdparty.ThirdPartyFileChecksumRepository


One fix is to add @Param("<paramName">) in front of each method parameter in these classes, I've verified this with one class.